packages/features/ee/teams/components/MemberInvitationModal.tsx

```diff
@@ -74,17 +76,16 @@ export default function MemberInvitationModal(props: MemberInvitationModalProps)
     enabled: !!session.data?.user?.org,
   });
 
+  const validateUniqueInviteMutation = trpc.viewer.teams.validateUniqueInvite.useMutation();
+
   // Check current org role and not team role
   const isOrgAdminOrOwner =
     currentOrg &&
     (currentOrg.user.role === MembershipRole.OWNER || currentOrg.user.role === MembershipRole.ADMIN);
 
-  const canSeeOrganization = !!(
-    props?.orgMembers &&
-    props.orgMembers?.length > 0 &&
-    currentOrg?.isPrivate &&
-    isOrgAdminOrOwner
-  );
```

Comment:
I think `currentOrg?.isPrivate` was a mistake here as only private org owners were able to see 'organizer' importMode. we should also allow normal org owners/admins to view `organizer` importMode